docs/docs/guide/04-blog/00-build-blog.md

```diff
@@ -6,534 +6,302 @@ slug: /guide/blog
-	// Get the context
-	ctx := sdk.UnwrapSDKContext(goCtx)
-
-	// highlight-start
-	// Create variable of type Post
-	var post = types.Post{
-		Creator: msg.Creator,
-		Title:   msg.Title,
-		Body:    msg.Body,
-	}
-
-	// Add a post to the store and get back the ID
-	id := k.AppendPost(ctx, post)
-
-	// Return the ID of the post
-	return &types.MsgCreatePostResponse{Id: id}, nil
-	//highlight-end
-}
+ignite chain serve
 ```
 
-## Define the `Post` type and `AppendPost` keeper method
-
-When you define the `Post` type in a proto file, Ignite CLI (with the help of
-`protoc`) takes care of generating the required Go files.
-
-Create the `proto/blog/blog/post.proto` file and define the `Post` message:
+Once the blockchain is running, we can use the binary to interact with it and
+see how the code handles creating, updating, and deleting blog posts. We can
+also see how it processes and responds to queries. This will give us a better
+understanding of how our application works and allow us to test its
+functionality.
 
-```protobuf title="proto/blog/blog/post.proto"
-syntax = "proto3";
+Here is an example of how to use the binary to create a new blog post on the
+blockchain:
 
-package blog.blog;
-
-option go_package = "blog/x/blog/types";
-
-message Post {
-  string creator = 1;
-  uint64 id = 2;
-  string title = 3;
-  string body = 4;
-}
 ```
-
-The contents of the `post.proto` file are standard. The file defines:
-
-* A package name `blog.blog` that is used to identify messages
-* The Go package `go_package = "blog/x/blog/types"` where new files are
-  generated
-* The message `message Post`
-
-Continue developing your blog chain.
-
-### Define keeper methods
-
-The next step is to define the `AppendPost` keeper method.
-
-Create the `x/blog/keeper/post.go` file and start thinking about the logic of
-the function and what you want to call the prefixes. The file will be empty for
-now.
-
-* To implement `AppendPost` you must first understand how the key store works.
-  You can think of a store as a key-value database where keys are
-  lexicographically ordered. You can loop through keys and use `Get` and `Set`
-  to retrieve and set values based on keys. To distinguish between different
-  types of data that a module can keep in its store, you can use prefixes like
-  `product/` or `post/`.
-
-* To keep a list of posts in what is essentially a key-value store, you need to
-  keep track of the index of the posts you insert. Since both post values and
-  post count (index) values are kept in the store, you can use different
-  prefixes: `Post/value/` and `Post/count/`.
-
-Then, add these prefixes to the `x/blog/types/keys.go` file:
-
-```go title="x/blog/types/keys.go"
-package types
-
-const (
-	// ...
-
-	// Keep track of the index of posts
-	PostKey      = "Post/value/"
-	PostCountKey = "Post/count/"
-)
+blogd tx blog create-post 'Hello, World!' 'This is a blog post' --from alice
```

Comment:
Is it worth mentioning that this command must be executed in an other terminal, in order to preserve the blockchain execution ?

Reply:
https://github.com/ignite/cli/pull/3234/commits/7ebfac825a1c01779c06ba5e8f8ca1fa800b1bad